db/Pos/DB/Functions.hs

```diff
@@ -128,6 +141,9 @@ rocksPutBytes k v DB {..} = Rocks.put rocksDB rocksWriteOpts k v
 rocksPutBi :: (Bi v, MonadIO m) => ByteString -> v -> DB -> m ()
 rocksPutBi k v = rocksPutBytes k (encodeStrict v)
 
+rocksPutStore :: (Store v, MonadIO m) => ByteString -> v -> DB -> m ()
```

Comment:
`Store` doesn't support streaming without additional overhead. Using a strict `ByteString` is faster.